website/docs/src/main/tut/api/chisel/latest/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-redirect_to: https://javadoc.io/doc/org.chipsalliance/chisel_2.13
+redirect_to: https://s01.oss.sonatype.org/service/local/repositories/snapshots/archive/org/chipsalliance/chisel_2.13/6.0.0-M3+55-6dbbc73c-SNAPSHOT/chisel_2.13-6.0.0-M3+55-6dbbc73c-SNAPSHOT-javadoc.jar/!/index.html
```

Comment:
This is technically a change as this was pointing at M3 before? However, I like this change. 👍

Reply:
Yeah true, but now it's more correct as latest, although maybe it should be titled "latest SNAPSHOT"